guides/plugins/plugins/storefront/add-icons.md

```diff
@@ -0,0 +1,71 @@
+# Add custom icons
+
+## Overview
+
+In order to design a outstanding and customized User Experience, it's important to think about icons, ideally 
```

Comment:
```suggestion
In order to design an outstanding and customized User Experience, it's important to think about icons, ideally 
```
I'm not sure about that whole sentence tbh. Maybe straight to the point like "This guide will explain how to add custom icons to your theme" or something similar.

Reply:
Or: In this guide you will learn how to use the icon renderer component as well as adding custom icons.